### PR TITLE
[dev] Bug AB#59185 [Mobile][My Ministries > Serving Opps] After adding a new Opportunity in My Ministry, the "Actions" pop out drawer does not close automatically | Extend Mobile Action Bar Actions Button and Actions Drawer to provide the option of toggling the Actions Drawer closed when an option is clicked

### PR DESCRIPTION
**Pull request by cletux87**
Execute onDrawerToggle instead of passing to parent component

### CODE REVIEW

**Review by groberts314 — APPROVED** [review]

**Comment by groberts314** [comment]
This is needed for HC Admin 1.44.0 (per Proposed Release Version on [Bug AB#59185](https://dev.azure.com/saddlebackchurch/Church%20Management/_workitems/edit/59185).

File: src/surfaces/actionBar/actionBarActionsButtonDrawerOption.jsx
Comment (groberts314) [inline src/surfaces/actionBar/actionBarActionsButtonDrawerOption.jsx:64]: Are the Boolean return values actually meaningful?
If not, I would like to see them all changed to just `return;` where we need to bail out early, and then remove the final one on Line 68.

Thoughts?

_**Not a Blocker**_

**Review by groberts314 — APPROVED** [review]
Looks good to me.
One question/possible improvement with respect the `return`s in `onParentClick()` but I have designated it a NAB.

File: src/surfaces/actionBar/actionBarActionsButtonDrawerOption.jsx
Comment (groberts314) [reply src/surfaces/actionBar/actionBarActionsButtonDrawerOption.jsx:64]: Yeah, let's please do that, if you would be so kind.

Comment (cletux87) [reply src/surfaces/actionBar/actionBarActionsButtonDrawerOption.jsx:64]: Sure

Comment (cbrewer1689) [reply src/surfaces/actionBar/actionBarActionsButtonDrawerOption.jsx:64]: The change looks good, but are we waiting for return change before merging this?

**Review by cbrewer1689 — APPROVED** [review]

**Comment by groberts314** [comment]
> are we waiting for return change before merging this?

If @cletux87 can bang that out pretty quickly, I think we should.

File: src/surfaces/actionBar/actionBarActionsButtonDrawerOption.jsx
Comment (cletux87) [reply src/surfaces/actionBar/actionBarActionsButtonDrawerOption.jsx]: Ready :)

File: src/surfaces/actionBar/actionBarActionsButtonDrawerOption.jsx
Comment (groberts314) [inline src/surfaces/actionBar/actionBarActionsButtonDrawerOption.jsx:64]: We don't need this one.  Functions automagically return when we get to the end! =)

**Comment by groberts314** [comment]
@cletux87 Looks good, although we technically we have one `return;` too many.  The last one can be elided.

File: src/surfaces/actionBar/actionBarActionsButtonDrawerOption.jsx
Comment (groberts314) [reply src/surfaces/actionBar/actionBarActionsButtonDrawerOption.jsx:64]: Actually nm.  This is not the end of the function.  Sorry; don't mind me.

**Comment by groberts314** [comment]
@cbrewer1689 I say :shipit:

File: src/surfaces/actionBar/actionBarActionsButtonDrawerOption.jsx
Comment (cletux87) [reply src/surfaces/actionBar/actionBarActionsButtonDrawerOption.jsx]: Not sure if I understand the return change @cbrewer1689  I can refactor the function to avoid using return. is that what you mean?

Comment (groberts314) [reply src/surfaces/actionBar/actionBarActionsButtonDrawerOption.jsx]: @cletux87 I think @cbrewer1689 was referring to exactly the work you did in [e248d81](https://github.com/saddlebackdev/react-cm-ui/pull/359/commits/e248d81b3a627cf2528f88147ddfd9fb98c2a0e0).  So, it's done!  I think we can merge this if @cbrewer1689 agrees.

**Pull request merged**
